requirements/pip.txt

```diff
@@ -30,9 +30,9 @@ pyyaml==6.0
 Pygments==2.10.0
 
 # Basic tools
-redis==3.5.3
-kombu==5.1.0
-celery==5.1.2
+redis==4.0.0
```

Comment:
this version was released 2 days ago, and 4.0.1 was released today, I think we should wait a little more before updating this one.